drivers/ipm/Kconfig

```diff
@@ -6,6 +6,21 @@ menuconfig IPM
 	  Include interrupt-based inter-processor mailboxes
 	  drivers in system configuration
 
+
+config IPM_QUARK_SE
+	bool "Quark SE IPM driver"
+	depends on IPM
+	help
+	  Driver for Quark SE mailboxes
+
+config IPM_QUARK_SE_MASTER
+	bool "Quark SE IPM master controller"
+	depends on IPM_QUARK_SE
+	help
+	  Enable this for the first CPU that initializes IPM.
+	  Sets up the initial interrupt mask and clears out all
+	  channels. Should be turned on for one CPU only.
+
```

Comment:
What's this about? I don't think we've had Quark SE support in the tree for quite awhile.

Reply:
Yep, just get rid of all Quark-related changes @kapi-no

Reply:
Something went really wrong in the rebase

Reply:
Yes, this is leftover from rebasing